configurations/Nisqually.json

```diff
@@ -55,7 +55,13 @@
         "OR",
         "com.ibm.ipzvpd.VINI({'CC': [50, 69, 50, 70]})",
         "OR",
-        "com.ibm.ipzvpd.VINI({'CC': [50, 69, 52, 48]})"
+        "com.ibm.ipzvpd.VINI({'CC': [50, 69, 52, 48]})",
```

Comment:
you make my life hard by not just telling me the hex/ascii values for these, or why they were added :O

Reply:
I had plans to comment on the ASCII to hex to decimal to ...